pmd-java/src/main/java/net/sourceforge/pmd/lang/java/JavaLanguageModule.java

```diff
@@ -19,12 +19,12 @@ public JavaLanguageModule() {
         super(NAME, null, TERSE_NAME, JavaRuleChainVisitor.class, "java");
         addVersion("1.3", new JavaLanguageHandler(3), false);
         addVersion("1.4", new JavaLanguageHandler(4), false);
-        addVersions(new JavaLanguageHandler(5), false, "1.5", "5");
-        addVersions(new JavaLanguageHandler(6), false, "1.6", "6");
-        addVersions(new JavaLanguageHandler(7), false, "1.7", "7");
-        addVersions(new JavaLanguageHandler(8), false, "1.8", "8");
-        addVersions(new JavaLanguageHandler(9), false, "9", "1.9");
-        addVersions(new JavaLanguageHandler(10), false, "10", "1.10");
+        addVersion("1.5", new JavaLanguageHandler(5), false, "5");
```

Comment:
I think we could improve further by hiding the `isDefault` parameter. There's only one default version, but all those calls have to mention this `false` argument, which is obscure unless you know what it is, or are in an IDE. We could just have `addVersion` for non-default versions, and a separate `addDefaultVersion` method

Reply:
interesting… I also wonder if we should overwrite defaults if one is already set, or complain about it…

Reply:
done